src/help/help_impl.cpp

```diff
@@ -1029,7 +1029,7 @@ void generate_unit_sections(const config* /*help_cfg*/, section& sec, int /*leve
 		for (const std::string &variation_id : type.variations()) {
 			// TODO: Do we apply encountered stuff to variations?
 			const unit_type &var_type = type.get_variation(variation_id);
-			const std::string topic_name = var_type.type_name() + "\n" + var_type.variation_name();
+			const std::string topic_name = var_type.variation_name();
```

Comment:
Tested the edge-case that a variation name is blank, and the UI still looks reasonable (it just displays a line with the icon, so the page is still accessible).